bubble_sort.rb

```diff
@@ -0,0 +1,28 @@
+def bubble_sort(arr)
+  loop do
+    swapped = false
+    (arr.length - 1).times do |i|
+      if arr[i] > arr[i + 1]
+        arr[i], arr[i + 1] = arr[i + 1], arr[i]
+        swapped = true
+      end
+    end
+    break if swapped == false
+  end
+  print arr
```

Comment:
🤔 I guess you were debugging this and forgot to remove the `print` statement, so the function is not returning `arr`.

> Build a method #bubble_sort that takes an array and __returns__ a sorted array.